cmake/modules/RootNewMacros.cmake

```diff
@@ -1449,4 +1449,27 @@ function(find_python_module module)
    set(PY_${module_upper}_FOUND ${PY_${module_upper}_FOUND} PARENT_SCOPE)
 endfunction()
 
+#----------------------------------------------------------------------------
+# Generate headers and link them through argparse method
```

Comment:
Could you elaborate and add that this routine create the command line options in a string format and imports them inside of ROOT.